include/secutils/credentials/store.h

```diff
@@ -164,12 +176,18 @@ X509_STORE* STORE_load_trusted(const char* files, OPTIONAL const char* desc, OPT
  * @param trust_dir   directory where to search for certificates
  * @param desc        description of trusted certs to use for error reporting, or null
  * @param recursive   if true, use recursive search in subdirectories
+ * @param vpm verification parameters; unless null, this enables checks whether
+ * all loaded certs are CA certs and are currently in their validity period,
+ * depending on 509_V_FLAG_USE_CHECK_TIME and X509_V_FLAG_NO_CHECK_TIME
```

Comment:
shouldn't `509_V_FLAG_USE_CHECK_TIME` be `X509_V_FLAG_USE_CHECK_TIME` instead?

Reply:
Good spotting - this was a copy error; fixed here and for all other occurrences.

On this occasion also improved compatibility with OpenSSL 1.x
and did three further small fixes/improvements.